### PR TITLE
Merge PR https://github.com/cocos2d/cocos2d-x/pull/3117

### PR DESCRIPTION
**Pull request by dumganhar**

### CODE REVIEW

**Comment by dumganhar** [comment]
@jotel 
I got 

```
07-16 16:54:40.325: D/cocos2d-x debug info(15495): OpenGL error 0x0501 in /Users/james/Project/cocos2d-x/samples/Cpp/TestCpp/proj.android/../../../../cocos2dx/draw_nodes/CCDrawingPrimitives.cpp lazy_init 100
07-16 16:54:40.325: D/cocos2d-x debug info(15495): OpenGL error 0x0501 in /Users/james/Project/cocos2d-x/samples/Cpp/TestCpp/proj.android/../../../../cocos2dx/draw_nodes/CCDrawingPrimitives.cpp lazy_init 102
```

when i tested this issue. Any ideas?

**Comment by dumganhar** [comment]
I found the reason and solution. I will update this PR.

**Comment by jotel** [comment]
cool... what was it?

Jaro

**Comment by dumganhar** [comment]
Please refer to my latest two commits: dabb66e and 0932a7c

File: cocos2dx/draw_nodes/CCDrawNode.cpp [hunk 1]
```diff
@@ -118,6 +120,10 @@ static inline Tex2F __t(const Vertex2F &v)
     ccGLBindVAO(0);
     _vao = 0;
 #endif
+    
+#if CC_ENABLE_CACHE_TEXTURE_DATA
+    NotificationCenter::getInstance()->removeObserver(this, EVNET_COME_TO_FOREGROUND);
```
Comment (nicolasgramlich) [inline cocos2dx/draw_nodes/CCDrawNode.cpp:125]: Typo: `EVNET_`. Seems like the constant itself has the typo.

Comment (minggo) [inline cocos2dx/draw_nodes/CCDrawNode.cpp:125]: Yep, good catch.

**Pull request merged**